pkg/runner/docker/docker.go

```diff
@@ -271,3 +278,35 @@ func (d *docker) PullImage(containerImageConfig *core.ContainerImageConfig, r *c
 	}
 	return nil
 }
+
+func (d *docker) WriteLogs(ctx context.Context, r *core.RunnerOptions) error {
```

Comment:
do we need to export this? Otherwise, lowercase it.

Reply:
Created internal function.